clang/unittests/Tooling/RecursiveASTVisitorTests/DeductionGuide.cpp

```diff
@@ -58,7 +58,7 @@ Bar s(1);
       DeductionGuideVisitor::Lang_CXX2a));
 }
 
-TEST(RecursiveASTVisitor, DeductionGuideImplicitMode) {
+TEST(RecursiveASTVisitor, DISABLED_DeductionGuideImplicitMode) {
   DeductionGuideVisitor Visitor(/*ShouldVisitImplicitCode*/ true);
   Visitor.ExpectMatch("Foo(T) -> Foo<int>", 11, 1);
   Visitor.ExpectMatch("Bar(type-parameter-0-0) -> Foo<int>", 14, 1);
```

Comment:
I think the fix should be straightforward, just adjust the content here. I will fix it afterwards.

Reply:
thanks! yea i just wasn't confident that i'd make the "right" change here if i just updated the expectation

Reply:
I just landed https://github.com/llvm/llvm-project/commit/2ef12b55b2a655262b03204581076522261d5eb3, you can remove the `DISABLED_` now.

Reply:
done, thanks!